### ISSUE TITLE
Fix linting scripts in fxa-payments-server and fxa-settings packages

### ISSUE THREAD

**Issue by pdehaan** [mozilla/fxa#11728]
## Description

Currently the **fxa-payments-server** and **fxa-settings** packages point to an invalid "lint:sass" script which breaks <kbd>yarn lint</kbd>.

### Steps to reproduce

Run <kbd>yarn lint</kbd> from the two package directories above and you should get the following error(s):

> _ERROR: Task not found: "lint:sass"_

### Expected result

No error/message.

### Actual result

> _ERROR: Task not found: "lint:sass"_

### Environment

Node 17.4.0

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXA-4481)

**Comment by StaberindeZA** [comment]
Hi @pdehaan 

Thank you for raising this issue. I've been keeping track of the different linter/prettier issues you've raised and looking forward to seeing the different outcomes.

For the `fxa-payments-server` we have the following issue raised to fix the script and the warnings/errors that the working linter script raises. (From what I remember there where quite a few)

#11546

**Comment by pdehaan** [comment]
@StaberindeZA Thanks for the heads-up! I added a comment in https://github.com/mozilla/fxa/issues/11546#issuecomment-1020483422 which links back here and possible solution.

TLDR: Should we merge this PR now since it solves the _immediate_ bug, and then submit a new ticket for investigating and adding in the Sass linting (based on how other FxA monorepo packages lint Sass so we can use consistent rules/packages)?

**Comment by cknowles-admin** [comment]
FxA is no longer synchronizing all issues between Jira and Github.  We are closing open issues and will selectively synchronize in the future.
